x-pack/plugins/observability_solution/observability/server/routes/types.ts

```diff
@@ -24,8 +24,9 @@ export interface ObservabilityRouteHandlerResources {
 }
 
 export interface ObservabilityRouteCreateOptions {
-  tags: string[];
+  tags?: string[];
   access?: 'public' | 'internal';
+  security?: RouteSecurity;
```

Comment:
this should be a top-level property of the route definition

Reply:
Thanks @dgieselaar 
I didn't see that update.. 
I updated the code.